src/models/generation/thermal_generation.jl

```diff
@@ -139,6 +139,6 @@ end
 ThermalGenSeason(; name = "init",
                 status = false,
                 bus = Bus(),
-                tech = nothing,
-                econ = nothing,
-                scalingfactor = TimeSeries.TimeArray(today(), [1.0])) = ThermalGenSeason(name, status, bus, tech, econ, scalingfactor)
+                tech = TechThermal(),
+                econ = EconThermal(),
+                scalingfactor = TimeArray(collect(DateTime(today()):Hour(1):DateTime(today()+Day(1))), ones(25))) = ThermalGenSeason(name, status, bus, tech, econ, scalingfactor)
```

Comment:
This default scaling factor creates 24 steps by default, there is ni need for that.

Reply:
I needed more than 1 step to measure the time delta, I can change it to have just 2 steps if you want